src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,131 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import cn from 'classnames';
+import { Person } from '../types';
+import { useSearchParams } from 'react-router-dom';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+import { FilterType } from '../types/FilterType';
+import { Params } from '../types/Params';
+import { FilterPeople } from '../types/FilterPeople';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const searchPeople = (relativeName: string | null) => {
+    return people.find(person => person.name === relativeName);
+  };
+
+  const [searchParams] = useSearchParams();
+
+  const sort = searchParams.get(Params.sort) || null;
+  const order = searchParams.get(Params.order) || null;
+  const query = searchParams.get(Params.query) || '';
+  const sex = searchParams.get(Params.sex) || FilterPeople.ALL;
+  const centuries = searchParams.getAll(Params.centuries || []);
```

Comment:
The `getAll` method should be used directly on `Params.centuries` without the logical OR operator. It should be `searchParams.getAll(Params.centuries)` to correctly retrieve all century parameters.